source/frontend/src/features/properties/map/project/add/models.ts

```diff
@@ -9,15 +9,30 @@ export class ProjectForm {
   projectStatusType?: string;
   region?: NumberFieldValue;
   summary?: string;
+  rowVersion?: number;
 
-  toApi(id: number = 1): Api_Project {
+  toApi(): Api_Project {
     return {
-      id: id,
+      id: this.id,
       code: this.projectNumber,
       description: this.projectName,
       projectStatusTypeCode: toTypeCode<string>(this.projectStatusType),
       regionCode: this.region ? toTypeCode<number>(+this.region) : undefined,
       note: this.summary,
+      rowVersion: this.rowVersion,
     };
   }
+
+  static fromApi(model: Api_Project): ProjectForm {
```

Comment:
If this has a to and from api we can add a test for isomorphism (ie input -> output -> input is the same)

Reply:
Added to UpdateProjectContainer.test.tsx